definitions/variable/electricitysources_tags.yaml

```
@@ -0,0 +1,22 @@
+# This file adds needed variables for the EMPIRE data upload
+-  Active Power|Electricity|{Electricity Source}:
+     description: Active power generated from {Electricity Source}
+     unit: MWh
+- Investment|Energy Supply|Electricity|{Electricity Source}:
+    description: Investment in {Electricity Source}
+    unit: billion USD_2010/yr
+- Variable Cost|Electricity|{Electricity Source}:
+    description: Variable cost for generating electricity from {Electricity Source} with the fuel cost
+    unit: EUR/MWh
+- Capacity|Electricity|{Electricity Source}:
+    description: Installed (available) capacity of {Electricity Source}
+    unit: GW
+- Capital Cost|Electricity|{Electricity Source}:
+    description: Capital cost for {Electricity Source}
+    unit: USD_2010/kW
+- Emissions|CO2|Electricity|{Electricity Source}:
```

Please follow common-definitions conventions.
```suggestion
- Emissions|CO2|Energy|Supply|Electricity|{Electricity Source}:
```